unit_tests/UnitTests.d

```diff
@@ -905,3 +905,21 @@ void foo ();
 D", options));
 
 }
+
+// Test global attributes
+unittest
+{
+    Options options;
+    options.globalAttributes = ["nothrow", "@nogc"];
+
+    assertTranslates(q"C
+void foo();
+C", q"D
+extern (C):
+nothrow:
+@nogc:
```

Comment:
Wouldn't it be better to have a syntax from the issue #111 , you've originally proposed?

```
extern (C) nothrow @nogc:
```